### PR TITLE
[READY] add blog about LS workshop

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Recap blog about the LS workshop in Zwolle by @IwannaDJ

### CODE REVIEW

**Comment by tarzan** [comment]
Don't we want to position ourselves as THE 'to-go-to-authority' on knowledge building and sharing? I think we could use these blog-posts to position our opinion and vision a bit more strongly. Would really like our audience to think: 'those guys (and girls) at Defacto, they really are onto something!'

File: source/blog/nl/2016-12-20-waardevolle-kennis-delen-met-learningspaces.html.markdown [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+LearningSpaces bleek de oplossing te zijn voor die regurgitatie van kennis en het vinden van kennis op de nieuw gefuseerde afdeling. Expertise wordt met LearningSpaces op een centraal punt met elkaar gedeeld en de naald in de enorme ‘kennishooiberg’ is makkelijker te vinden.
+
+## "Het gaat zo eenvoudig!"
+
+Wanneer het tijd is voor alle vijfentwintig aanwezigen om aan de slag te gaan, maken ze als oefening een Space over hun eigen hobby. Spontaan ontstaat er een eerste Space over chocola, een ander gaat over schaatsen. Foto's en links vanaf YouTube worden in de hoofdstukken gesleept en er is positieve verbazing over het mooie eindproduct. Velen staan versteld van de simpele werking, gehoord langs de zijlijn: "Het werkt erg intuitief! Het gaat zo eenvoudig, ik denk dat het komt doordat er zo weinig knoppen zijn, heel fijn!"
+
+## Fans
+
+Mooie reactie op een product waar mijn collega's en ik zelf al fan van zijn. Bij Defacto gebruiken we het voor het delen van ál onze expertise. [Probeer LearningSpaces de eerste zestig dagen gratis](http://www.learningspaces.nl){:target="_blank"} en wordt ook fan ;).
```
Comment (fatboypunk) [inline source/blog/nl/2016-12-20-waardevolle-kennis-delen-met-learningspaces.html.markdown]: zestig mag je volgens mij als `60` schrijven

File: source/blog/nl/2016-12-20-waardevolle-kennis-delen-met-learningspaces.html.markdown [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+---
+
+Je kunt ook deelnemen aan één van onze workshops of een demo. Neem daarvoor contact op met [Iwanna de Jonge](mailto:iwanna@defacto.nl) of bel met Defacto op 050 31 44 832.
+
+<!-- De middag begint met een praatje van Marcel Horlings, developer van LearningSpaces, over de visie achter het product. Hij legt uit wat de bestaansreden van LearningSpaces is: *"We hebben veel [zwaardere LMS applicaties gezien](http://blog.learningspaces.io/ending-the-lms-era-for-organisations/){:target="_blank"} en dachten dat dit makkelijker kon. Plus, gaan we nog wel op dezelfde manier met kennis om over vijftien jaar?"* LearningSpaces ziet hij als de simpelste manier om e-learning te maken, maar het is meer: *"Het is ook een goede manier om inzichtelijk te maken welke kennis er binnen een organisatie is. En daarnaast een snelle manier om kennis met elkaar te delen."*
```
Comment (fatboypunk) [inline source/blog/nl/2016-12-20-waardevolle-kennis-delen-met-learningspaces.html.markdown]: So i Guess we can remove this right?

File: source/blog/nl/2016-12-20-waardevolle-kennis-delen-met-learningspaces.html.markdown (hunk 1 above)
Comment (dkhgh) [inline source/blog/nl/2016-12-20-waardevolle-kennis-delen-met-learningspaces.html.markdown]: regurgitatie. regurgitatie zelfst.naamw. [medisch] terugstroming (meestal van voedsel uit maag naar mond)

File: source/blog/nl/2016-12-20-waardevolle-kennis-delen-met-learningspaces.html.markdown (hunk 2 above)
Comment (RonaldDijkstra) [inline source/blog/nl/2016-12-20-waardevolle-kennis-delen-met-learningspaces.html.markdown]: 💩  oops

File: source/blog/nl/2016-12-20-waardevolle-kennis-delen-met-learningspaces.html.markdown (hunk 1 above)
Comment (RonaldDijkstra) [inline source/blog/nl/2016-12-20-waardevolle-kennis-delen-met-learningspaces.html.markdown]: :nauseated_face: 

We should use something else here.

**Comment by RonaldDijkstra** [comment]
- Removed some unnecessarily content 
- Changed publish date to today
- Changed the sentence with regurgitatie a little 

Please approve my changes

**Comment by RonaldDijkstra** [comment]
👍

**Pull request merged**